src/shared/conf-parser.c

```diff
@@ -648,6 +649,36 @@ int config_parse_bool(const char* unit,
         return 0;
 }
 
+int config_parse_id128(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        sd_id128_t t, *result = data;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = sd_id128_from_string(rvalue, &t);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse 128bit ID/UUID, ignoring: %s", rvalue);
+        } else if (sd_id128_is_null(t)) {
```

Comment:
as per coding style, we generally don't place {} around single line if blocks:

formatting

Reply:
Sorry, overlooked this one...